frontend/webpack.common.js

```diff
@@ -203,3 +204,33 @@ const getHtmlWebpackPlugin = (env, appName, dirName) => {
 const isGitAvailable = () => {
   return fs.existsSync(path.join(__dirname, "..", ".git"));
 };
+
+const addThymeleafNoncePlaceholder = (e) => {
+  e.attributes["th:attr"] = "nonce=${cspNonce}";
+}
+
+class CSPNoncePlaceholderInjectorPlugin {
+  apply(compiler) {
+    compiler.hooks.compilation.tap(
+      "CSPNoncePlaceholderInjectorPlugin",
+      (compilation) => {
+        HtmlWebpackPlugin.getHooks(compilation).alterAssetTags.tapAsync(
+          "CSPNoncePlaceholderInjectorPlugin",
+          (data, cb) => {
+            const { scripts, styles, meta } = data.assetTags;
+            scripts.forEach(addThymeleafNoncePlaceholder);
+            styles.forEach(addThymeleafNoncePlaceholder);
+            meta.push(
+              HtmlWebpackPlugin.createHtmlTagObject("meta", {
+                name: "csp-nonce",
+                "th:attr": "content=${cspNonce}",
+              })
+            );
+
+            cb(null, data);
+          }
+        );
+      }
+    );
+  }
+}
```

Comment:
Tämä muutos injektoi näitä uusia tageja tietysti myös AKT:n ja OTR:n index.html:ään. Niistä ei sinänsä pitäisi olla haittaa vaikka niiden palvelinpäässä ei vielä injektoidakaan CSP-noncea - varmistan kuitenkin tämän vielä.